deep_regression.py

```diff
@@ -0,0 +1,251 @@
+from __future__ import absolute_import, division, print_function
+
+import tensorflow as tf
+import matplotlib.pyplot as plt
+import numpy as np
+import scipy.io as sio
+import pandas as pd
+from sklearn.utils import shuffle
+from sklearn.model_selection import train_test_split
+import tensorflow.contrib.slim as slim
+from tensorflow.contrib.framework.python.ops.variables \
```

Comment:
It's ok to have import lines longer than 80 chars. As far as I know you are advised not to use \ for line continuation.